docs/templates/modules.md

```diff
@@ -1,8 +1,12 @@
-# Template inheritance
+# Reusing template code
 
-In instances where you want to reuse code across different Coder templates, such as common scripts or resource definitions, we suggest using [Terraform Modules](https://developer.hashicorp.com/terraform/language/modules).
+To reuse code across different Coder templates, such as common scripts
+or resource definitions, we suggest using [Terraform
+Modules](https://developer.hashicorp.com/terraform/language/modules).
 
-These modules can be stored externally from Coder, like in a Git repository or a Terraform registry. Below is an example of how to reference a module in your template:
+You can store these modules externally from your Coder deployment,
+like in a git repository or a Terraform registry. This example shows
+how to reference a module from your template:
 
 ```hcl
```

Comment:
@marcpaq thoughts on adding an example to be used from https://registry.coder.com/modules

Reply:
Excellent idea. I recommend adding this in another PR. This current PR is already pretty busy.

What do you think?

Reply:
Sure. We can do the registry in a different PR.